scipy/stats/_morestats.py

```diff
@@ -3932,22 +3935,22 @@ def directionalmean(samples, *, axis=0, normalize=True):
 
     See also
     --------
-    circmean: directional mean in two dimensions for circular data
+    circmean: circular mean; i.e. directional mean for 2D *angles*
```

Comment:
I wanted to emphasize that `circmean` is the function to use when the observations are expressed as *angles*